src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+
+* {
```

Comment:
point 9 in checklist:
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/3080ef38-5865-4d9a-92e6-ff75524f31d4)